src/config/wallets.ts

```diff
@@ -123,7 +123,7 @@ export const supportWalletObj = {
     isSupportMobileApp: true,
   },
   [SupportWallet.SubWalletNative]: {
-    img: require('/src/assets/img/logo-subwallet.svg'),
+    img: require('/src/assets/img/logo-subwallet.png'),
```

Comment:
Logo is more than 100KB. I am wandering if it could be smaller (in order to improve loading performance) since we are showing him on the portal in 24 x 24 px size

Reply:
Okay!I'll check others too and change the size